src/client/pages/Download/components/AppImage.tsx

```diff
@@ -6,33 +6,50 @@ type Props = {
   width: number
 }
 
-const Image = styled.img<Props>`
-  width: ${({ width }) => width}px;
+const Image = styled.img<{ imageWidth: number }>`
+  width: ${({ imageWidth }) => imageWidth}px;
   max-width: 100%;
   height: auto;
 `
 
-const getImageSrc = (currentLocale: string) => {
+const getImageProps = (currentLocale: string) => {
   const basePath = '/new-member-assets/download'
 
   switch (currentLocale) {
     case 'se':
-      return `${basePath}/hedvig-app-se.png`
+      return {
+        src: `${basePath}/hedvig-app-se.png`,
+        width: 1660,
+        height: 1400,
+      }
     case 'no':
-      return `${basePath}/hedvig-app-no.png`
+      return {
+        src: `${basePath}/hedvig-app-no.png`,
+        width: 1660,
+        height: 1400,
+      }
     case 'dk':
-      return `${basePath}/hedvig-connect-payment-dk.png`
+      return {
+        src: `${basePath}/hedvig-connect-payment-dk.png`,
```

Comment:
Maybe since we're in here anyway we should take the opportunity to change names of this and `hedvig-connect-payment-dk-en.png` to `hedvig-app-dk` and `hedvig-app-dk-en`..?